cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperatorTest.java

```diff
@@ -78,6 +80,10 @@ public void testCreateCluster(TestContext context) {
         when(mockDcOps.scaleUp(anyString(), anyString(), anyInt())).thenReturn(Future.succeededFuture(42));
         when(mockDcOps.scaleDown(anyString(), anyString(), anyInt())).thenReturn(Future.succeededFuture(42));
 
+        ArgumentCaptor<ConfigMap> metricsCaptor = ArgumentCaptor.forClass(ConfigMap.class);
+        ArgumentCaptor<String> metricsNameCaptor = ArgumentCaptor.forClass(String.class);
+        when(mockCmOps.reconcile(anyString(), metricsNameCaptor.capture(), metricsCaptor.capture())).thenReturn(Future.succeededFuture(ReconcileResult.created(null)));
```

Comment:
If you're not using the captured metrics and metrics names then there's no point using a captor. You could just use `any()`, `anyString()` etc.